server/routes/customer.js

```diff
@@ -123,7 +123,6 @@ customerRouter.post('/create', (req, res) => {
           })
           .catch((err) => {
```

Comment:
your .catch is in the wrong spot. it should be one level up to match with the first .then